indico/modules/rb/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -378,6 +383,12 @@ class BookingEditForm extends React.Component {
             disabled={submitSucceeded}
           />
         </Segment>
+        {renderPluginComponents('rb-form-extra_fields', {
```

Comment:
```suggestion
        {renderPluginComponents('rb-booking-form-extra-fields', {
```

better name and no ugly mix between dashes and underscores